src/aktualizr_lite/test_list.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+set -e
+
+build_dir=$(pwd)
```

Comment:
This is a bit brittle. Usually ctest runs tests from the root project directory, so it doesn't look so true.
Because you only use `build_dir` to  access `aktualizr-lite`, it could as well be directly given as an argument like `akrepo_bin`.